test/distributed/_composable/fsdp/test_fully_shard_training.py

```diff
@@ -346,6 +349,7 @@ def test_non_root_forward_backward(self):
         self.assertEqual(ref_nonroot_loss, nonroot_loss)
         self.assertEqual(ref_model(inp).sum(), model(inp).sum())
 
+    @test_compiled_fsdp()
     def test_multi_forward_module(self):
```

Comment:
I realized I missed a `@skip_if_lt_x_gpu(2)` for this test. Could you sneak it into this PR 😅

Reply:
will add it to the PR